docs/manual/installation/install-contao.de.md

```diff
@@ -45,7 +45,7 @@ Sobald der Contao Manager alle Pakete installiert hat, musst du das [Contao-Inst
 aufrufen um die Datenbank zu aktualisieren.
 
 
-## Installation über die Kommandozeile {#installation-ueber-die-kommandozeile}
+## Installation über die Kommandozeile
```

Comment:
```suggestion
## Installation über die Kommandozeile
```

Why did you remove the anchor? Doesn't this change the anchor? If yes, have you checked any references to this anchor?